src/commands/moderation/faq.ts

```diff
@@ -82,7 +82,7 @@ const faqEmbeds = [
 export const faq: CommandDefinition = {
     name: 'faq',
     description: 'Sends the FAQ',
-    requiredPermissions: ['BanMembers'],
+    requirements: { permissions: ['BanMembers'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to ADMIN & MODERATOR roles?